subsys/bluetooth/mesh/Kconfig

```diff
@@ -663,6 +663,15 @@ config BT_MESH_ACCESS_DELAYABLE_MSG_CHUNK_COUNT
 
 endif # BT_MESH_ACCESS_DELAYABLE_MSG
 
+config BT_MESH_DELAYABLE_PUBLICATION
+	bool "Delayable publication"
+	default y
+	help
+	  When enabled, the periodic publications are randomly delayed by 20 to 50ms. Publications
```

Comment:
I guess it is worth to add that the feature is optional.

Reply:
I'm not sure if this is needed as otherwise this won't be togglable at all

Reply:
Will it be convenient for user to know it is possible to disable without breaking of qualification?

Reply:
With that logic every option should have such clarification.

Reply:
I assumed the only optional by spec.

Reply:
We don't have this anywhere else (except newly added delayable access layer messages).